lib/index.js

```diff
@@ -83,17 +70,19 @@ function setupRateLimitEventExt(server, options) {
         }
 
         const error = Boom.tooManyRequests();
-        error.output.headers = new RateLimitHeaders(
-          newMinimumLimitResponse.limit,
-          newMinimumLimitResponse.remaining,
-          newMinimumLimitResponse.reset);
+        if (options.addHeaders) {
```

Comment:
As mentioned previously, I would not make this optional unless there is a good reason to